v0.3/cruorg/components/input/input.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+function output_form($form_type, $input_types, $backgroundBlack = false)
+{
+?>
+  <div <?php if ($backgroundBlack) echo 'style="background-color: #383F43; padding-bottom: 8px"' ?>>
```

Comment:
@sebamelzi Don't include HTML for design preview purposes in the functions. Only AEM markup should be in these.